Externals/libusb/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+add_library(usb SHARED EXCLUDE_FROM_ALL
+    libusb/core.c
+    libusb/core.c
+    libusb/descriptor.c
+    libusb/hotplug.c
+    libusb/io.c
+    libusb/strerror.c
+    libusb/sync.c
+)
+set_target_properties(usb PROPERTIES VERSION 1.0.19)
+target_include_directories(usb
+    # turns out other projects also have "config.h", so make sure the
+    # LibUSB one comes first
+    BEFORE
+
+    PUBLIC libusb
+    PRIVATE "${CMAKE_CURRENT_BINARY_DIR}"
+)
+
+if(WIN32 OR CYGWIN)
+    target_sources(usb PRIVATE libusb/os/windows_usb.c)
+    set(OS_WINDOWS TRUE)
+elseif(APPLE)
+    target_sources(usb PRIVATE libusb/os/darwin_usb.c)
+    find_library(COREFOUNDATION_LIBRARY CoreFoundation)
+    find_library(IOKIT_LIBRARY IOKit)
+    find_library(OBJC_LIBRARY objc)
+    target_link_libraries(usb PRIVATE
+        ${COREFOUNDATION_LIBRARY}
+        ${IOKIT_LIBRARY}
+        ${OBJC_LIBRARY}
+    )
+    set(OS_DARWIN TRUE)
+elseif(ANDROID)
```

Comment:
In the spirit of the other Externals' CMakeLists, I've commented out this section of code instead of deleting it. Let me know if you'd like an approach other than that.